### DISCUSSION TITLE
2022.08.17 MSF Delegates Meeting 11:00 EDT / 15:00 UTC

### DISCUSSION THREAD

**Discussion by funksoup** [omigroup/MSF-Delegates#27] [General]
Tentative agenda:

- Miro jam session (45 minutes)
review draft of MSF delegate participation, then general brainstorming (eg: diagram ecosystem/participation/community strategies)

- Gather agenda points for next meeting (5 minutes)

**Comment by futuristudios** [comment]
conversation was video recorded. here's the [file on ipfs](https://ipfs.io/ipfs/QmSgzVoA7HK7kGvGDA8hznqLm4rG7yuyzYzijHiDT6GkMX), could someone with permissions please uploaded to [OMI youtube channel](https://www.youtube.com/channel/UCB3huP95JrOTma5LfiDCd4A).
